### PR TITLE
Ignore some files in event system (prevents crashes)

### PR DESCRIPTION
**Pull request by marc-mrt**

### CODE REVIEW

File: lib/EventSystemEpic/ProjectFeature/Model/ignore.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+const nodeModules = "node_modules/";
+const tmpDir = "tmp/";
+const tmpExt = ".tmp";
+const gitExt = ".git";
+
+const paths = [nodeModules, tmpDir, tmpExt, gitExt];
+
+export const ignoreFilePaths = (path: ?string) =>
+  paths.reduce((res, test) => !(path && path.includes(test)), true);
```
Comment (GauthierPLM) [inline lib/EventSystemEpic/ProjectFeature/Model/ignore.js]: Why using a reducer when you can use array.include ?

```suggestion
  paths.includes(path);
```

File: lib/EventSystemEpic/ProjectFeature/Model/ignore.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+"use babel";
+// @flow
+
+const nodeModules = "node_modules/";
+const tmpDir = "tmp/";
+const tmpExt = ".tmp";
+const gitExt = ".git";
+
+const paths = [nodeModules, tmpDir, tmpExt, gitExt];
```
Comment (GauthierPLM) [inline lib/EventSystemEpic/ProjectFeature/Model/ignore.js]: Why not putting the values directly into the array ?

File: lib/EventSystemEpic/ProjectFeature/Model/ignore.js (hunk 1 above)
Comment (marc-mrt) [reply lib/EventSystemEpic/ProjectFeature/Model/ignore.js]: I want to test whether or not the content of the path includes one of the values in paths, what you're suggesting doesn't work

File: lib/EventSystemEpic/ProjectFeature/Model/ignore.js (hunk 2 above)
Comment (marc-mrt) [reply lib/EventSystemEpic/ProjectFeature/Model/ignore.js]: C L A R I T Y

**Comment by marc-mrt** [comment]
I'll do some more work tomorrow, i'll update this PR

**Pull request merged**